test/inductor/test_max_autotune.py

```diff
@@ -255,14 +255,239 @@ def mm(a, b):
             Y = mm(a, b)
             torch.testing.assert_close(Y_compiled, Y)
 
+    def _test_max_autotune_cutlass_backend_simple_fusion(
+        self,
+        dynamic: bool = False,
+        max_autotune_gemm_backends: str = "CUTLASS",
+        mixed_precision=False,
+        fp16=True,
+    ):
+        """
+        Test simple fusion that's not covered by specific lowering
+        """
+
+        if max_autotune_gemm_backends == "CUTLASS" and torch.version.hip:
+            return
+
+        torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction = (
+            mixed_precision
+        )
+
+        def mm(a, b):
+            return (a @ b) * 3.0
+
+        # Note: The ops that are available
+        # also depend on the alignment of the shapes
+        # so if these shapes don't all align to at least 8 elements
+        # it can happen that no Cutlass 3.x op is available
+        # that allows fusions
+        a = torch.randn(256, 32).cuda()
+        b = torch.randn(32, 256).cuda()
+        if fp16:
+            a = a.half()
+            b = b.half()
+
+        with config.patch(
+            {
+                "max_autotune": True,
+                "autotune_in_subproc": False,
+                "max_autotune_gemm_backends": max_autotune_gemm_backends,
+                "cuda.cutlass_dir": _CUTLASS_DIR,
+                "cuda.cutlass_max_profiling_configs": 4,
+                "cuda.cutlass_only_evt_capable_ops": True,
+                "cuda.version": "12.2",  # required to enable the Kernels we need
```

Comment:
Yes, it's necessary here, otherwise the Kernels we need for testing won't be listed at all. It does not mean that these Kernels are performant when CUDA < 12.2, but at least they seem to exist, and tests are working.

Reply:
Yeah, I understand that the value is passed to the CUTLASS generator which won't generate the EVT kernels under 12.2 (although, I'd expect under 12.1). I was just trying to make sure that the kernels are functioning. Perf doesn't matter at this point. Thanks for confirming.

Reply:
@kadeng I think 12.1 should be enough. Just checked https://github.com/NVIDIA/cutlass/blob/main/python/cutlass_library/generator.py, only found rules for 12.1 instead of 12.2.